hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestHttp.java

```diff
@@ -199,45 +208,50 @@ public void testInvalidRequestPath() throws Exception {
     rawSpan = normalizer.convert("tenant-key", span);
 
     assertEquals("/", HttpSemanticConventionUtils.getHttpPath(rawSpan.getEvent()).get());
-    assertEquals(
-        rawSpan.getEvent().getHttp().getRequest().getPath(),
-        HttpSemanticConventionUtils.getHttpPath(rawSpan.getEvent()).get());
   }
 
   @ParameterizedTest
   @MethodSource("provideMapForTestingRequestUserAgentTagKeysPriority")
-  public void testRequestUserAgentTagKeysPriority(Map<String, String> tagsMap) throws Exception {
+  public void testRequestUserAgentTagKeysPriority(Map<String, String> tagsMap, String userAgent)
+      throws Exception {
 
     Span span = createSpanFromTags(tagsMap);
     RawSpan rawSpan = normalizer.convert("tenant-key", span);
 
-    assertEquals(
-        rawSpan.getEvent().getHttp().getRequest().getUserAgent(),
-        HttpSemanticConventionUtils.getHttpUserAgent(rawSpan.getEvent()).get());
+    // now, we are not populating first class fields. So, it should be null.
+    assertNull(rawSpan.getEvent().getHttp());
```

Comment:
Isn't this repetitive/irrelevant in every test ? Maybe we can have another test just checking nullity of first class fields?

Reply:
I kept it after - `RawSpan rawSpan = normalizer.convert("tenant-key", span);` - convert call.